ethereum/rlp/src/main/java/tech/pegasys/pantheon/ethereum/rlp/AbstractRLPOutput.java

```diff
@@ -141,6 +141,10 @@ public int encodedSize() {
     return payloadSizes[0];
   }
 
+  public int uncheckedEncodedSize() {
```

Comment:
Would be helpful to add some javadoc here.

Reply:
Done.